assimilation_code/modules/observations/forward_operator_mod.f90

```diff
@@ -323,8 +323,6 @@ subroutine get_obs_ens_distrib_state(ens_handle, obs_fwd_op_ens_handle, &
    ! each ensemble member into global_qc_value
    allocate(var_istatus(qc_ens_handle%num_copies))
 
-   if (.not. isprior) call put_single_copy(obs_fwd_op_ens_handle, OBS_GLOBAL_QC_COPY, prior_qc_copy)
```

Comment:
Great catch finding this, it has been in there a while.

A note for future refactoring, this is where 'storing the prior copy' came in 2015: 
https://github.com/NCAR/DART/commit/6905705937e5313886dacf037811876b627e3645